src/components/Drawer/Drawer.tsx

```diff
@@ -14,6 +15,8 @@ export type Props = {
   setExpanded: (v: boolean) => void;
   /** The menu items to be displayed in the drawer */
   menuItems: MenuItem[];
+  /** Render prop function to display something over the Navigation */
+  renderHeader?: () => React.ReactNode;
```

Comment:
You can change the type to ```renderHeader?: React.FC```

Reply:
That would mean `renderHeader` returns a function. From what I udnderstand this is following the render props pattern, so renderHeader is in fact returning a `ReactNode` or a `ReactElement` (objects) and not a function (`FC`)

Reply:
render props patterns requires children prop as function to work properly.

Since this is just a JSX.Element prop. @kboutsika You can just add a type of `JSX.Element`or `ReactNode` without `() =>`. Like we did in the Button for example https://github.com/Orfium/orfium-ictinus/blob/master/src/components/Button/Button.tsx#L33.

Reply:
yes thank you both for the review! @michalisKout this is just another approach that for me its also pretty fine! However it more like personal preference if you you want to use a render prop or a JSX.Element.